examples/expect-extend/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "private": true,
+  "version": "0.0.0",
+  "name": "example-expect-extend",
+  "devDependencies": {
+    "@babel/core": "*",
+    "@babel/preset-env": "*",
+    "@babel/preset-typescript": "*",
+    "@jest/globals": "*",
+    "babel-jest": "*",
+    "expect": "*",
+    "jest": "*"
```

Comment:
do we need `workspace:*` here so we resolve to the versions we want?

Reply:
potentially just `resolutions` in root: https://github.com/facebook/jest/blob/60eb4160b5258eed5faf381e0c8ea6662076cc6c/package.json#L157-L159